functionary/train/training_utils.py

```diff
@@ -72,114 +121,78 @@ def initialize_tokenizer(
 
 
 def preprocess_logits_for_metrics(logits, labels, tokenizer_size):
-    """Preprocesses the logits during evaluation by computing the greedy token predictions for
-    accuracy calculation and loss values for perplexity calculation. Both pred_ids and loss are
-    of shape (batch_size x seq_len)"""
-
```

Comment:
We should keep docstring . If you think its obsolete in terms of information, better to update it, rather than removing completely.